modules/040-node-manager/webhooks/validating/node_group

```diff
@@ -172,7 +172,21 @@ EOF
     return 0
   fi
 
-    cat <<EOF > "$VALIDATING_RESPONSE_PATH"
+  # Only update operation checks
+  operationType="$(context::jq -r '.review.request.operation')"
+  if [[ "${operationType}" == "UPDATE" ]]; then
+    # Forbid changing nodeType
+    newNodeType="$(context::jq -r '.review.request.object.spec.nodeType')"
+    oldNodeType="$(context::jq -r '.review.request.oldObject.spec.nodeType')"
+    if [[ "${oldNodeType}" != "${newNodeType}" ]]; then
+        cat <<EOF > "$VALIDATING_RESPONSE_PATH"
+{"allowed":false, "message":"it is forbidden to change nodeType for NodeGroup"}
```

Comment:
Maybe message should be like `.spec.nodeType field is immutable` to be closed with native k8s messages